en/getting_started/px4_basic_concepts.md

```diff
@@ -30,7 +30,7 @@ In order to set up, control and interact with your PX4 drone you need to connect
 
 * [Remote Control \(RC\) Connection](../getting_started/rc_transmitter_receiver.md) - Connection between the RC handset and vehicle-based receiver that you use to manually direct the vehicle.
 * [Data/Telemetry Connection](TBD) - Connection between QGroundControl and your drone by Data Radio, Wifi or USB cable.
-* Off-board Connection - A data connection between PX4 and external microcomputer that can control PX4.
+* Off-board Connection - A data connection via serial cable or wifi between PX4 and external microcomputer that can control PX4.
```

Comment:
Strictly speaking the data connection can be any mechanism at all. I think this is a helpful change though.